chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/assaults/MemoryAssault.java

```diff
@@ -141,4 +161,15 @@ private void waitUntil(int ms) {
             }
         }
     }
+
+    /**
+     * Java 8 or lower: 1.6.0_23, 1.7.0, 1.7.0_80, 1.8.0_222
+     * Java 9 or later: 9.0.1, 11.0.4, 12, 12.0.1
+     *
+     * @return true if Java <= 8
+     */
+    private static boolean isJava8OrLower() {
```

Comment:
I like that you moved that to a method, even tough I don't see the need for it. What I don't like is that you increased the matching range, even tough wer're not supporting java versions below 8. So I would suggest keep the method (maybe rename to `isJavaVersion8()`) and match specifically for 1.8 java versions.

Reply:
I am going to change it to Java 8 only. But in this case I preferr a variable again, so I will change it (but only local scope of the method)

Reply:
I just changed it, check it out 👍